content/bd/0003-er-model.md

```diff
@@ -135,16 +135,16 @@ Podemos aplicar ambas as restrições simultaneamente.
 Membro -> (Sócio, Não Sócio) | Gestor | (Regular, Não Regular)
 Sócio -> (Sócio Bronze, Sócio Silver, Sócio Gold)
```

Comment:
Is this meant to be here?

Reply:
No :sweat_smile: 

Forgot to remove the draft